conda_libmamba_solver/state.py

```diff
@@ -394,8 +395,14 @@ def channels_from_specs(self) -> Iterable[Channel]:
 
     def maybe_free_channel(self) -> Iterable[Channel]:
         # conda 25.9 removes context.restore_free_channel
-        if getattr(context, "restore_free_channel", False):
-            yield Channel.from_url("https://repo.anaconda.com/pkgs/free")
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                message="conda.base.context.Context.restore_free_channel",
+                category=DeprecationWarning,
+            )
```

Comment:
While `getattr` protects us from failing once `context.restore_free_channel` is removed we also need to ignore the warning itself in this scenario.